blob/azureblob/azureblob_test.go

```diff
@@ -0,0 +1,58 @@
+package azureblob
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"testing"
+	
+	"github.com/google/go-cloud/blob/driver"
+	"github.com/google/go-cloud/blob/drivertest"
+	"github.com/google/go-cloud/internal/testing/setup"
+)
+
+const (
+	bucketName = "go-cloud-bucket"
+)
+
+type harness struct {
+	settings Settings
+	closer   func()
+}
+
+func newHarness(ctx context.Context, t *testing.T) (drivertest.Harness, error) {
+	p, done := setup.NewAzureTestPipeline(ctx, t)
+	accountName := os.Getenv("AZURE_STORAGE_ACCOUNT_NAME")
```

Comment:
Correct, I did see that. Just my misunderstanding with the test harness. I did fix this now, it will replay without the environment variables. The account name is set to a constant to avoid the mismatch URL exceptions.